govcd/vapp.go

```diff
@@ -81,7 +81,7 @@ func (v *VApp) Refresh() error {
 	return nil
 }
 
-func (v *VApp) AddVM(orgvdcnetworks []*types.OrgVDCNetwork, vapptemplate VAppTemplate, name string) (Task, error) {
+func (v *VApp) AddVM(orgvdcnetworks []*types.OrgVDCNetwork, vapptemplate VAppTemplate, name string, acceptalleulas bool) (Task, error) {
```

Comment:
@hodgesrm , this is one of the changes that I wanted to propose from the beginning.
We should be passing structures rather than positional parameters.